### PR TITLE
py-pyaudio: cleaned py-pyaudio Portfile

### PR DESCRIPTION
**Pull request by Jakker**
* Added github handle
* Removed md5 checksum and added size

#### Description

See also conversation at PR 2476 

###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G65
Xcode 9.4.1 9F2000

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?

### CODE REVIEW

File: python/py-pyaudio/Portfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ set my_name             PyAudio
 version                 0.2.11
 categories-append       audio
 platforms               darwin
-maintainers             NLnetLabs.nl:jaap
+maintainers             {@Jakker NLnetLabs.nl:jaap}
```
Comment (pmetzger) [inline python/py-pyaudio/Portfile]: Would it also be okay to add `openmaintainer` at the end?

Comment (Jakker) [reply python/py-pyaudio/Portfile]: Oops, I actually originally planned to do that and then apparently forgot. Need a new commit for that?

Comment (yan12125) [reply python/py-pyaudio/Portfile]: An amended commit is fine for such trivial changes.

Comment (Jakker) [reply python/py-pyaudio/Portfile]: Did an amended commit. Hope I did it right (I'm a git newbie).

Comment (yan12125) [reply python/py-pyaudio/Portfile]: Well, looks like the change is not in. You can check overall changes at https://github.com/macports/macports-ports/pull/2486/files

Comment (Jakker) [reply python/py-pyaudio/Portfile]: Apologies for the noise, but the change is now in.

File: python/py-pyaudio/Portfile [hunk 2]
```diff
@@ -23,9 +23,9 @@ master_sites            pypi:P/PyAudio/
 
 distname                ${my_name}-${version}
 
-checksums               md5     7e4c88139284033f67b4336c74eda3b8 \
-                        rmd160  7a6bb88f56622555e77eb799e4ee74ff970b6e92 \
-                        sha256  93bfde30e0b64e63a46f2fd77e85c41fd51182a4a3413d9edfaf9ffaa26efb74
+checksums               rmd160  7a6bb88f56622555e77eb799e4ee74ff970b6e92 \
+                        sha256  93bfde30e0b64e63a46f2fd77e85c41fd51182a4a3413d9edfaf9ffaa26efb74 \
+                        size 37428
```
Comment (yan12125) [inline python/py-pyaudio/Portfile]: The number after `size` should be aligned. For example
```
checksums               rmd160  7a6bb88f56622555e77eb799e4ee74ff970b6e92 \
                        sha256  93bfde30e0b64e63a46f2fd77e85c41fd51182a4a3413d9edfaf9ffaa26efb74 \
                        size    37428
```

**Comment by yan12125** [comment]
Also, remember to squash all commits into a single one after fixing everything.

**Comment by pmetzger** [comment]
@Jakker you needed to squash the changes, but sadly instead the branch got a bit messed up and all the commits are still there. However, as the changes were easy, I've applied them to the master directly. I would suggest that as git is now a pretty universal tool, you might want to read https://git-scm.com/book/en/v2 — it's worthwhile.

**Pull request closed without merging**